### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by zacharyweston** [zacharyweston/codelab-css-transitions-animations-transforms#1]
Hey @chrisvfritz, can you take a look at this? It's [hosted here](http://thankful-harbor.surge.sh/) and meets the following [project](http://lansingcodelab.com/lessons/css-transitions-animations-transforms/1) criteria:
- [x] Build a table using [a `table` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#Examples), listing statistics for various branches of a national organization. Example stats should all be very impressive business terms, such as "Synergy Coefficient" or "Projected Bitcoin Dynamics". Include at least 3 stats for a very impressive table.
- [x] When the user hovers over each table cell, the background color should slowly **transition** to a different color.
- [x] When the user hovers over the table, it should **transition** to be slightly larger with a fancy **scale transform**.
- [x] When the user hovers over a table row (`tr`), the row should **transition_ to the right slightly with a __translateX transform**.
- [x] The client needs a fancy logo, so quickly create one with the [Hipster Logo Generator](https://www.hipsterlogogenerator.com/), then include it in your page, above the table. It's very important to the client that a super awesome **keyframe animation** is applied when a user hovers over the logo. (You can create your own or steal [an attention seeker from animate.css](https://github.com/daneden/animate.css/tree/master/source/attention_seekers)).

**Comment by chrisvfritz** [comment]
Nice _Better Off Ted_ reference. :smiley: A few notes:

### Transition on `table` scale

When hovering over the `table`, it looks like it grows larger, but it snaps to that size rather than slowly transitioning. See if you can fix that.

### Consistent browser prefixes

In order to [support Internet Explorer 9 and older iOS and Android](http://caniuse.com/#feat=transforms2d), let's use the same vendor prefixes you're [using here](https://github.com/zacharyweston/codelab-css-transitions-animations-transforms/blob/master/style.css#L9-L11) for `transform`.

In case you're curious, the reason browser prefixes aren't included in animate.css is because they're using a tool called `autoprefixer`. It automatically add prefixes where you need to for the browsers you want to support. We'll be diving into that tool in a later lesson, but for now, we need to add the prefixes manually.

## Removing `.DS_Store` files from git

It looks like a `.DS_Store` file snuck into this repository. That's a hidden file that OS X automatically generates in folders managed by Finder. Since they have more to do with your computer than the project though, let's remove them from the project.

Check out [this question on Stack Overflow](http://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository) for instructions on banishing those `.DS_Store` files forever!

**Comment by zacharyweston** [comment]
.DS_Store file has been removed, extra browser prefixes have been removed, and the table hover transition has been added.  New URL is succinct-minute.surge.sh

**Comment by chrisvfritz** [comment]
YES! Looks great now. :ship: it (aka :shipit:). :stuck_out_tongue:
